README.md

```diff
@@ -47,9 +47,8 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
     airflow db init
     ```
 
-   > **Note:** `AIRFLOW__CORE__ENABLE_XCOM_PICKLING` no longer needs to be enabled from `astro-sdk-python` release 1.2 and above. This functionality is now deprecated as our custom xcom backend handles serialization.
-    Users can either use a custom XCom backend [`AstroCustomXcomBackend`](https://astro-sdk-python.readthedocs.io/en/latest/guides/xcom_backend.html) with Xcom pickling disabled (or) enable Xcom pickling.
-    Currently, custom XCom backends are limited to data types that are json serializable. Since Dataframes are not json serializable, we need to enable XCom pickling to store dataframes.
+   > **Note:** `AIRFLOW__CORE__ENABLE_XCOM_PICKLING` no longer needs to be enabled from `astro-sdk-python` release 1.3.3 with airflow >= 2.5 and above. This functionality is now deprecated as Airflow xcom backend handles serialization.
```

Comment:
what about users who use airflow < 2.5? The `README.md` needs to have a compatibility matrix with what config needs to be enabled , with what env variables and for which version of SDK and which version of airflow. Otherwise it is not clear currently from the docs.

Reply:
Yup, that makes sense, maybe we can point to read the doc link instead of posting it here?

Reply:
I would prefer it to be a table here to show a quick summary to the user

Reply:
Okay, works!